src/shared/extensionUtilities.ts

```diff
@@ -85,7 +86,9 @@ export function getIdeProperties(): IdeProperties {
  * Returns whether or not this is Cloud9
  */
 export function isCloud9(): boolean {
-    return getIdeType() === IDE.cloud9
+    const settings = new DefaultSettingsConfiguration('aws-debug')
```

Comment:
Why create a new group?

Reply:
Partially to make it more hidden, partially because the current key `'aws'`  is in the constants file which would create a circular dependency. I can use `'aws'` if you'd rather though, no strong opinions.

Reply:
+1 for 'aws' hardcoded. Adding a new group is cognitive overhead for developers, IMO, with no benefit AFAICT